src/astro/files/locations/gcs.py

```diff
@@ -0,0 +1,37 @@
+from typing import Dict, List
+from urllib.parse import urlparse, urlunparse
+
+from astro.constants import FileLocation
+from astro.files.locations.base import Location
+from astro.utils.dependencies import gcs
+
+
+class GCS(Location):
+    """Handler GS object store operations"""
+
+    location_type = FileLocation.GS
+
+    def get_transport_params(self) -> Dict:
+        """get GCS credentials for storage"""
+        if self.conn_id:
+            gcs_hook = gcs.GCSHook(self.conn_id)
+            client = gcs_hook.get_conn()  # type: ignore
+        else:
+            hook = (
+                gcs.GCSHook(gcp_conn_id=self.conn_id) if self.conn_id else gcs.GCSHook()
+            )
+            client = hook.get_conn()
```

Comment:
This does not make sense.. isn't it just the following:

```suggestion
        hook = gcs.GCSHook(gcp_conn_id=self.conn_id) if self.conn_id else gcs.GCSHook()
        client = hook.get_conn()
```